### PR TITLE
Add description for LacpTimer parameter

### PR DESCRIPTION
**Pull request by velkovb**

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Looks good.

File: docset/windows/netlbfo/set-netlbfoteam.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Accept wildcard characters: False
 ```
 
 ### -LacpTimer
-{{Fill LacpTimer Description}}
+How often inter-connected devices exchange LACP protocol data units (PDUs) or control messages.
```
Comment (JohanFreelancer9) [inline docset/windows/netlbfo/set-netlbfoteam.md]:
```suggestion
Specifies how often inter-connected devices exchange LACP protocol data units (PDUs) or control messages.
```

**Comment by rmca14** [comment]
No Acrolinx or staging link.

**Pull request merged**